src/wallets/Wallets.tsx

```diff
@@ -17,6 +20,27 @@ export const Wallets = (): JSX.Element => {
     rendererLog.debug(`walletStatus: ${walletState.walletStatus}`)
   }, [walletState.walletStatus])
 
+  const {translateError} = useTranslation()
+  useEffect(() => {
+    pipe(
+      walletState.error,
+      option.map((error) => translateError(error)),
+      option.fold(
+        () => void 0,
+        (error) => {
+          message.error(error, 10, () => {
+            switch (walletState.walletStatus) {
+              case 'INITIAL':
+              case 'LOADED':
+              case 'LOADING':
+                walletState.refreshSyncStatus()
```

Comment:
Should we provide the refresh function also here when it's always refreshing in the background anyway?

Reply:
Refreshing was stopping on error and was stuck on `loading` state if refresh wasn't triggered here

Reply:
At least if the error happened during the initial load

Reply:
After changes from #48 It should work quite nicely together I think

Reply:
yeah, that's also what I'm thinking. It's coming together :D